lib/hdb/hdb.asn1

```diff
@@ -52,7 +52,9 @@ HDBFlags ::= BIT STRING {
 	force-canonicalize(30),		-- force the KDC to return the canonical
 					-- principal irrespective of the setting
 					-- of the canonicalize KDC option
-	do-not-store(31)		-- Not to be modified and stored in HDB
+	do-not-store(31),		-- Not to be modified and stored in HDB
+	explicit-kvno-changes-only(32)	-- New keys only become effective on
```

Comment:
We have a number of unused members between 19 and 29, inclusive.  Maybe use one of those?

Reply:
And if we stick to 32, make sure load/dump preserves this new flag and that the LDAP HDB backend uses 64-bit integers for the principal attributes field or else does not support this flag.  (Also, do LDAP servers support 64-bit integers?)

Reply:
Integers in LDAP are unbounded precision. Although it was common on 32bit servers to only implement them as 32bits, the ASN.1 specification does not limit their size at all. And 64bits is certainly safe on 64bit servers.

Reply:
Thanks @hyc.  I knew that LDAP supports large integers, but I didn't know if *servers* (or even client libraries) would.  In any case, not getting support for this flag in the event of truncation is not fatal, and the user can check for this.

Reply:
Argh, I forgot to check what the kadmin protocol does in Heimdal:

```
        krb5_store_int32(sp, princ->attributes);
```

Reply:
Well, I'll use `(20)`, as that's available and I'm already using `1<<20` for this elsewhere in this PR.

Reply:
You could use the higher bits for force-canonicalise and do-not-store, as they should never be stored.